core/src/main/java/io/github/kaktushose/jdac/JDACommands.java

```diff
@@ -114,8 +117,18 @@ public void shutdown() {
     }
 
     /// Updates all slash commands that are registered with [CommandScope#GUILD]
+    ///
+    /// This is equivalent to calling `updateGuildCommands(List.of());`
     public void updateGuildCommands() {
-        updater.updateGuildCommands();
+        updateGuildCommands(List.of());
+    }
+
+    /// Updates all slash commands that are registered with [CommandScope#GUILD]
+    ///
+    /// @param guilds a [Collection] of guilds to update. If an empty [Collection] is passed, will update all guilds
```

Comment:
empty list for all guilds is somehow unintuitive for me

Reply:
Same. What if I retrieve the guilds that shall be updated via some other function. Passing an empty collection should result in no guilds being updated.

Reply:
thought about it as some sort of filter. empty list -> filter passes for all. But because the scope of this PR is to reduce api calls, i agree that this design is not the best. What about just separating the methods?